x/orderbook/bucket.go

```diff
@@ -44,6 +50,35 @@ func marketIDindexer(obj orm.Object) ([]byte, error) {
 	return ob.MarketID, nil
 }
 
+// marketIDTickersIndexer produces in SQL parlance, a compound index
+// (MarketID, AskTicker, BidTicker) -> index
+func marketIDTickersIndexer(obj orm.Object) ([]byte, error) {
+	if obj == nil || obj.Value() == nil {
+		return nil, nil
+	}
+	orderbook, ok := obj.Value().(*OrderBook)
+
+	if !ok {
+		return nil, errors.Wrapf(errors.ErrState, "expected orderbook, got %T", obj.Value())
+	}
+
+	return BuildMarketIDTickersIndex(orderbook), nil
+}
+
+// BuildMarketIDTickersIndex indexByteSize = 8(MarketID) + ask ticker size + bid ticker size
+func BuildMarketIDTickersIndex(orderbook *OrderBook) []byte {
+	askTickerByte := make([]byte, tickerByteSize)
+	copy(askTickerByte, orderbook.AskTicker)
+
+	bidTickerByte := make([]byte, tickerByteSize)
+	copy(bidTickerByte, orderbook.BidTicker)
+
+	res := append(orderbook.MarketID, askTickerByte...)
```

Comment:
This is good.

You can also try the `bytes.Join()` function you mentioned in another PR review:

`return bytes.Join([][]byte{orderbook.MarketID, askTickerByte, bidTickerByte}, nil)`